app/src/data/departments.js

```diff
@@ -81,7 +81,7 @@ const departments = [
   {
     name: 'Planning & Zoning',
     description: 'Planning, preservation and design services',
-    learnMore: 'The Purpose of the Planning and Zoning Department is to provide planning, preservation and design services. PAZ is responsible for long range planning, current planning (including historic preservation, annexation, and zoning), urban design services, and updates of the Land Development Code. Current Projects underway: Imagine Austin, CodeNEXT, South Center Waterfront Initiative.',
+    learnMore: 'The purpose of the Planning and Zoning Department is to provide planning and zoning, preservation, and design services to make Austin the most livable city in the country\n\n Planning & Zoning is responsible for long range planning, current planning (including historic preservation, annexation, and zoning), urban design services, and updates of the Land Development Code.\n\n Current Projects underway include: Imagine Austin, CodeNEXT, South Center Waterfront Initiative.',
```

Comment:
`The purpose of the Planning and Zoning Department is to provide planning and zoning...`

Redundant language?